config/crd/bases/logging.openshift.io_apiauditlogfilters.yaml

```diff
@@ -0,0 +1,192 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.2
+  creationTimestamp: null
+  name: apiauditlogfilters.logging.openshift.io
+spec:
+  group: logging.openshift.io
+  names:
+    categories:
+    - logging
+    kind: APIAuditLogFilter
+    listKind: APIAuditLogFilterList
+    plural: apiauditlogfilters
+    singular: apiauditlogfilter
+  scope: Namespaced
```

Comment:
I'm playing @xperimental here... should this be cluster scoped?

Reply:
Gone in latest version - policy is all inside the 'filters' section of  ClusterLogForwarder resource, no separate resource.